src/sql/workbench/parts/jobManagement/browser/media/jobHistory.css

```diff
@@ -258,40 +304,182 @@ table.step-list tr.step-row td {
 	flex-direction: column;
 }
 
-jobhistory-component {
+jobhistory-component,
+notebookhistory-component {
 	display: flex;
 	flex-direction: column;
 	height: 100%;
 	width: 100%;
 }
 
-jobhistory-component > .jobhistory-heading-container {
+jobhistory-component > .jobhistory-heading-container,
+notebookhistory-component > .jobhistory-heading-container {
 	display: flex;
 }
 
-jobhistory-component > .jobhistory-heading-container > .icon.in-progress {
+jobhistory-component > .jobhistory-heading-container > .icon.in-progress,
+notebookhistory-component > .jobhistory-heading-container > .icon.in-progress {
 	width: 20px;
 	height: 20px;
 	padding-top: 16px;
 	padding-left: 20px;
 }
 
-jobhistory-component > .agent-actionbar-container {
+jobhistory-component > .agent-actionbar-container,
+notebookhistory-component > .agent-actionbar-container {
 	border-top: 3px solid #f4f4f4;
 }
 
-.vs-dark jobhistory-component > .agent-actionbar-container {
+.vs-dark jobhistory-component > .agent-actionbar-container,
+.vs-dark notebookhistory-component > .agent-actionbar-container {
 	border-top: 3px solid #444444;
 }
 
-.hc-black jobhistory-component > .agent-actionbar-container {
+.hc-black jobhistory-component > .agent-actionbar-container,
+.hc-black notebookhistory-component > .agent-actionbar-container {
 	border-top: 3px solid #2b56f2;
 }
 
-jobhistory-component .step-table.prev-run-list .monaco-tree-wrapper .monaco-tree-row {
+jobhistory-component
+	.step-table.prev-run-list
+	.monaco-tree-wrapper
+	.monaco-tree-row,
+notebookhistory-component
+	.step-table.prev-run-list
+	.monaco-tree-wrapper
+	.monaco-tree-row {
 	width: 96%;
 }
 
-jobhistory-component .agent-actionbar-container > .monaco-toolbar.carbon-taskbar {
+jobhistory-component
+	.agent-actionbar-container
+	> .monaco-toolbar.carbon-taskbar,
+notebookhistory-component
+	.agent-actionbar-container
+	> .monaco-toolbar.carbon-taskbar {
 	margin: 10px 0px 5px 0px;
 }
+.notebook-grid {
+	display: grid;
+	grid-template-columns: repeat(auto-fill, minmax(100px, 1fr));
+	grid-gap: 10px;
+	padding: 10px;
+}
+
+.notebook-grid-item {
+	border-radius: 5px;
+	padding-bottom: 5px;
+	height: 95px;
+	display: block;
+	border-radius: 0px;
+}
+
+.notebook-grid-item > img {
+	display: block;
+	margin-left: auto;
+	margin-right: auto;
+	margin-top: 10px;
+	width: 50px;
+	height: 50px;
+}
+
+.notebook-grid-item:hover {
+	background: #dcdcdc !important;
+}
+
+.vs-dark .notebook-grid-item:hover,
+.hc-black .notebook-grid-item:hover {
+	background: #444444 !important;
+}
+
+.notebook-grid-item > .img-success {
+	background-image: url(./NotebookSuccess_16x.svg);
+	background-position: center;
+	display: block;
+	margin-left: auto;
+	margin-right: auto;
+	margin-top: 10px;
+	width: 40px;
+	height: 40px;
+	background-size: 40px 40px;
+	background-repeat: no-repeat;
+}
+
+.vs-dark .notebook-grid-item > .img-success,
+.hc-black .notebook-grid-item > .img-success {
+	background-image: url(./NotebookSuccess_16x_white.svg);
```

Comment:
try running `yarn gulp vscode-win32-x64` and make sure all your svgs resolve correctly.